### ISSUE TITLE
Prebuilt Images

### ISSUE THREAD

**Issue by TheFrozenDuck** [linaro-swg/optee_android_manifest#76]
I have some problems with building AOSP+OPTEE (v3.13.0) and I would currently like to fall back on prebuilt images while weeding out the errors. I have seen that in `master-dirty` there was directory with prebuilt images linked. The latest built seems to be uploaded in May 2021 [1]. I  was wondering if I that built was a proper built for OPTEE+AOSP.

[1] https://people.linaro.org/~victor.chong/prebuilt/pie/3130/hikey960/

**Comment by vchong** [comment]
@TheFrozenDuck Yes, they're all proper builds. `master-dirty` was an attempt to build on upstream AOSP `master` which proved to be too much of a moving target [2] so we moved back to using our latest stable version which is `pie`. [1] above is OP-TEE v3.13 release on `pie`.

[2] The introduction of GKI [3] also made moving forward with the `master` branch difficult due to the lack of support of GKI in our trees.
[3] https://source.android.com/devices/architecture/kernel/generic-kernel-image
